.github/workflows/client-app.yml

```diff
@@ -85,24 +79,38 @@ jobs:
           workspaces: |
             apps/desktop/src-tauri
 
-      - name: Make App Binary
-        run: pnpm build:client
+      - name: Build Affine
```

Comment:
I think this can be removed. tauri build has already included the two in `beforeBuildCommand`.

Reply:
Yes, removed